slasher/db/kv/block_header.go

```diff
@@ -120,11 +120,11 @@ func (s *Store) PruneBlockHistory(ctx context.Context, currentEpoch, pruningEpoc
 	if pruneTill <= 0 {
 		return nil
 	}
-	pruneTillSlot := uint64(pruneTill) * params.BeaconConfig().SlotsPerEpoch
+	pruneTillSlot := params.BeaconConfig().SlotsPerEpoch.Mul(uint64(pruneTill))
```

Comment:
`pruneTillSlot` should be a number and I don't think the name really represents what it does.

Reply:
You are right, will change to number. As about naming -- I've tried not to touch anything not directly related to types, the delta is huge as it is.